### ISSUE TITLE
Search/Category filter for What We Do

### ISSUE THREAD

**Issue by matthewedwards12** [ustwo/ustwo.com-frontend#228]
Filter by categories similar to filtering on Blog
Blocked by #315

**Comment by matthewedwards12** [comment]
Matthew to review categories

**Comment by matthewedwards12** [comment]
What We Do category drop down picker. 

3 categories to use:

ustwo
work
venture

**Comment by matthewedwards12** [comment]
Phil/Matthew to talk to designer about colour and text choices for tags

**Comment by matthewedwards12** [comment]
Perhaps we could use the Events page selector like this:

Everything // Client Work // Ventures // Own Products

**Comment by phil-linnell** [comment]
Closed as reassigning to new issue tracker.
